### ISSUE TITLE
App bundle does not launch on macOS

### ISSUE THREAD

**Issue by mishaturnbull** [mishaturnbull/PySpeedTest#37]
### What were you trying to do when the error occurred?
Launch the program.

### Did you get an error message?  If so, what did it say?
No error message.  The program opens very briefly then closes immediately.

### Are you able to reproduce the problem?  If so, how?
Yes -- launch the program.

### Platform details:

Operating system: Mac virtual machine, hosted on Windows 10

Version: e.g. MacOS 10.12

Using the program as one file

Is this the first time you've used (or attempted to use) the program?  Yes.
  - If yes, is there a file called `config.ini` anywhere?  No.
  
### Any other details that may be relevant:
The executable can be called from console via:
```bash
./PySpeedTest_v2.0.0_mac.app/Contents/MacOS/PySpeedTest_v2.0.0_mac
```
and will run fine.  The file `config.ini` is created normally in this case.  The bundle still will not launch after this has been done.

**Comment by mishaturnbull** [comment]
Just to clarify, this issue is taking after allowing access to run the program through Preferences as described in https://mishaturnbull.github.io/PySpeedTest/macs.html.

**Comment by mishaturnbull** [comment]
Relevant: https://github.com/pyinstaller/pyinstaller/issues/1892 and https://github.com/pyinstaller/pyinstaller/issues/1804.  

Sounds like the app is being run in a separate, random folder.  Maybe `persistence.py` is not working as I think it is?

**Comment by mishaturnbull** [comment]
Should be fixed in 8638d1941ff64c5836f2e3c95373a23e3e08f49d, but I'm not going to mark it closed until confirmation.

**Comment by mishaturnbull** [comment]
Confirmed to work on multiple different Macs (and a VM).
